testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -199,8 +199,10 @@
 
 When(/^I include the recommended child channels$/) do
   toggle = "//span[@class='pointer']"
+  xpath = "//i[contains(@class, 'fa-toggle-off')]"
+
   if page.has_xpath?(toggle, wait: 5)
-    find(:xpath, toggle).click
+    find(:xpath, toggle).click if page.has_xpath?(xpath, wait: 5)
   end
 end
```

Comment:
That would be more elegant I guess, as otherwise if that timeout pass, you will just skip the command but it will not raise any kind of error and you will not notice it.

In any case, we have a default wait time, not sure if it's lower than 5.

```suggestion
  When(/^I include the recommended child channels$/) do
    toggle_off = "//i[contains(@class, 'fa-toggle-off')]"
    raise 'The toggle is not expanded' unless page.has_xpath?(toggle_off, wait: 5)
    
    toggle = "//span[@class='pointer']"
    if page.has_xpath?(toggle, wait: 5)
      find(:xpath, toggle).click
    end
  end
```

Reply:
We do want to skip it if it does not find it, otherwise it raises an error

Reply:
Basically, if the toggle is already on, we do not want to do anything

Reply:
Ah but if the toggle is missing completely it might be an issue, so it can still be adapted

Reply:
Maybe
```
toggle = "//span[@class='pointer']"
raise 'The toggle is not present' unless page.has_xpath?(toggle, wait: 5)
toggle_off = "//i[contains(@class, 'fa-toggle-off')]"
find(:xpath, toggle).click if page.has_xpath?(toggle_off, wait: 5)
```